src/Components/GameSession.js

```diff
@@ -28,28 +28,29 @@ export default function GameSession() {
   function handleTileClick(e) {
     let id = e.currentTarget.id;
     if (id !== null) {
```

Comment:
We can check here whether the tile is already daubered i.e `if (id !== null && !dauberedTiles[id])`

Reply:
I pushed changes. Code has the same effect as your suggestion just broken out into two if statements. Also, had to add a check for tile 12 to keep selected word count correct. Tested against hard-coded word list and my local dev API server.